pkg/client/didexchange/client_test.go

```diff
@@ -474,19 +475,24 @@ func TestServiceEvents(t *testing.T) {
 	newDidDoc, err := (&mockcreator.MockDIDCreator{}).Create("test")
 	require.NoError(t, err)
 
+	didService, err := service.GetPthid(newDidDoc.Service)
+	require.NoError(t, err)
+
 	request, err := json.Marshal(
 		&didexchange.Request{
 			Type:  didexchange.RequestMsgType,
 			ID:    id,
 			Label: "test",
+			Thread: &decorator.Thread{
```

Comment:
Currently all parent thread test cases refer to public did. Test cases should be mixture of invitations and public dids. You could use invitation here since this is what our first release is about.